src/test/java/com/contrast/labs/ai/mcp/contrast/ADRServiceTest.java

```diff
@@ -575,4 +702,51 @@ private List<Attack> createMockAttacks(int count) {
 
         return attacks;
     }
+
+    /**
+     * Creates mock ProtectData for testing
```

Comment:
somewhat of a superfluous comment.  It doesn't say anything that the code doesn't clearly state.  There are a lot of  these types of comments in this code base.  Once again, not a bit deal, just something I noticed.